### ISSUE TITLE
Enderchest should also be added for dimensional-inventories

### ISSUE THREAD

**Issue by KominoStyle** [Thomilist/dimensional-inventories#8]
Heya,

I just found out that Enderchest are not added in the mod, since i only know javascript and not java, i cant tell how hard it would be to add, but i think it shouldn't be a problem :), would be nice

we use the 1.20.2 Version

**Comment by Thomilist** [comment]
Hey again,

Are you talking about the vanilla ender chest? If so, that's a bit strange. It's been implemented since the first release, but maybe there's a bug or incompatibility in there somewhere.

I gave it a quick test, both in singleplayer and on a dedicated server. In both cases, ender chest functionality worked, but I also didn't have any other mods installed.

Could you send...
1. A copy of the `<world>/dimensionalinventories` folder (a ZIP archive is fine)
2. A list of mods and versions (this should be at the top of the log output when you start the server)

**Comment by KominoStyle** [comment]
Heya,

Um, we're really sorry, but I've got it sorted now. So, when I created the world, the enderchest got duplicated into the new dimension. Then, I configured the inventory on both ends, meaning both had the same enderchest setup.

Since we had our enderchest set up with our usual default settings, it always appeared like it was the same enderchest in both dimensions. We only noticed this while testing a building, and, well, we accidentally opened the enderchest.

Our bad for the false alarm!
